### ISSUE TITLE
Support Service Fabric remoting

### ISSUE THREAD

**Issue by bjorkstromm** [bjorkstromm/ServiceFabric.Grpc#2]

**Comment by fcastellscha** [comment]
Do you have any plans to implement this? We are using remoting extensively and it would be interesting to be able to replace the default remoting implementation with gRPC without code changes in the services and clients (except from the service bootsrapping, which in our case it's shared across all services).

**Comment by bjorkstromm** [comment]
@fcastellscha no, currently I’m not working on gRPC support in Service Fabric Remoting, as my focus have shifted away from Service Fabric Reliable Services and Actors. All I know is that it should be doable, but it’s quite tricky. See [Twitter thread](https://twitter.com/aelij/status/979732319534833664?s=21) for reference.

**Comment by fcastellscha** [comment]
OK, no problem. Thanks for the Twitter reference.
